src/commands/chat/chat.js

```diff
@@ -0,0 +1,51 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
```

Comment:
use the builder from the DiscordJS package
```js
const { SlashCommandBuilder } = require('discord.js')
```

Reply:
Understood 👍